files/root/menu

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+function load_kmap() {
+    for keymap in $(find /usr/share/keymaps/ -type f -name \*.bin); do
+        echo "${A:-0}. ${keymap:19}"; ((A+=1));B+=($keymap); 
+    done;
+    read -p "Your choice: " keymap
+    loadkmap < ${B[$keymap]}
+}
+
+function install_NextSec() {
+    if [ $(grep "${1##*/}" /proc/partitions | wc -l) -gt 1 ]; then
+       echo -e "Multiple partitions foundi on $1, proceed forcing install ?\n[All content on device wil be deleted]"
+       read -rp "Do you want to proceed? (y/n) " force
+       case $force in
+	    [yY] ) msg=$(install-nx.sh -f -t $1);;
+	    * ) msg="Aborted";;
+       esac
+    else
+       msg=$(install-nx.sh -t $1)
+    fi
+    echo -e "Install: \n$msg"
+}
+
+function perhaps_install_NextSec() {
+    for disk in $(find /dev/sd[a-z] | grep -v $(df -t vfat /boot | tail -n 1| cut -d " " -f 1| tr -d "1")); do
```

Comment:
This find will not work for disks not named `/dev/sd*`. 
Example: NVME are usually named `/dev/nvme*`.

Try to use `block info` or install `blkid`